app/components/Header.tsx

```diff
@@ -0,0 +1,176 @@
+  const [hamburger, setHamburger] = useState(false);
+
+  const pathname = usePathname();
+
+  const handleHamburger = () => {
+    setHamburger(!hamburger);
+  };
+
+  return (
+    <Stack>
+      <Grid templateColumns={"500px 1fr 100px"}>
+        <Box p="12px">
+          <Image
+            src="/images/kenworthy-logo.jpg"
+            alt="Kenworthy Machine"
+            width={500}
+            height={500}
+          />
+        </Box>
+        <Box
+          display="flex"
+          alignItems="center"
+          justifyContent="flex-end"
+          gap="24px"
+        >
+          <Link
+            href="/"
+            color="white"
+            fontSize="24px"
+            borderBottom={
+              pathname === "/" ? "solid var(--primary-red) 2px" : "none"
+            }
+            _hover={{
+              color: "var(--text-color-light)",
+              borderBottom: "solid var(--primary-red) 2px",
+              transitionDuration: "400ms",
+            }}
+          >
+            Home
+          </Link>
+
+          <Link
+            href="/request-quote"
+            color="white"
+            fontSize="24px"
+            borderBottom={
+              pathname === "/request-quote"
+                ? "solid var(--primary-red) 2px"
+                : "none"
+            }
+            _hover={{
+              color: "var(--text-color-light)",
+              borderBottom: "solid var(--primary-red) 2px",
+              transitionDuration: "400ms",
+            }}
+          >
+            Request a Quote
+          </Link>
+
+          <div onClick={handleHamburger}>
+            <Image
+              src="/images/hamburger-menu.png"
+              alt="Hamburger Menu"
+              width={40}
+              height={40}
+            />
+          </div>
+        </Box>
+        <Box></Box>
+      </Grid>
+      {hamburger && (
+        <Grid
+          templateColumns="repeat(5, 1fr)"
+          gap="24px"
+          mx="12px"
+          mb="12px"
+          borderRadius="50%"
+        >
+          <Link
+            href="/request-quote"
+            color="white"
+            backgroundColor="var(--primary-red)"
+            borderRadius="20px"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            padding="6px"
+            fontWeight="700"
+            _hover={{
+              color: "var(--text-color-light)",
+              transitionDuration: "300ms",
+            }}
+          >
+            Request a Quote
+          </Link>
+          <Link
+            href="/view-machines"
+            color="black"
+            backgroundColor="white"
+            borderRadius="20px"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            padding="6px"
+            fontWeight="700"
+            _hover={{
+              color: "var(--text-color-light)",
+              transitionDuration: "300ms",
```

Comment:
I was wondering if this might be a good candidate for [creating a custom variant](https://chakra-ui.com/docs/components/link/theming) for the extended navbar buttons/links potentially extending it to all the buttons as they seem to be having the same or very similar styling. It'll require a bit of refactoring, but the Header component will look much cleaner and we will be able to utilize the custom theme throughout the app

Reply:
Okay, I created a custom variant for the Links by using extendTheme. It may be the case that Archaa will incorporate this into a larger theme, but for now it is located in the new lib folder under the file name buttonVariants.